lua/lspconfig/server_configurations/circom-lsp.lua

```diff
@@ -0,0 +1,14 @@
+return {
+  default_config = {
+    cmd = { 'circom-lsp' },
+    filetypes = { 'circom' },
```

Comment:
`    root_dir = util.find_git_ancestor,`

Reply:
adding  `root_dir = util.find_git_ancestor,`

Reply:
- The root_dir is now set to util.find_git_ancestor, allowing the language server to determine the project's root directory based on Git repositories. This enhances project context when working within Git-managed projects.

- single_file_support is enabled with a value of true. This setting allows the language server to operate in the current directory when working with single Circom files, providing improved support for single-file projects.